spec/directives/function/whitespace.hrx

```diff
@@ -0,0 +1,172 @@
+<===> function/before_name/scss/input.scss
+@function
+  a() {}
+
+<===> function/before_name/scss/output.css
+
+<===>
+================================================================================
+<===> function/before_name/sass/input.sass
+@function
+  a()
+
+<===> function/before_name/sass/output.css
+
+<===>
+================================================================================
+<===> function/after_name/sass/input.sass
+@function a
+  ()
+
+<===> function/after_name/sass/output.css
+
+<===>
+================================================================================
+<===> return/before_value/scss/input.scss
+@function a() {@return 
+  b}
+
+<===> return/before_value/scss/output.css
+
+<===>
+================================================================================
+<===> return/before_value/sass/input.sass
+@function a()
+  @return
+    b
+
+<===> return/before_value/sass/output.css
+
+<===>
+================================================================================
+<===> nested_at_rule/debug/scss/input.scss
```

Comment:
I'm not sure there's a point in testing multiple different nested at rules. That just seems like combinatorially many tests for little value.